### PR TITLE
Exact Words Clock App

### PR DESCRIPTION
**Pull request by bmsleight**
This is a clock for expressing the time in exact words. Each minute of 
the day has a different phrase.

### CODE REVIEW

**Comment by bmsleight** [comment]
size of app-icon.js corrected.

**Comment by thyttan** [comment]
> Sorry - can not get ./bin/sanitycheck.js to run locally. No experience with node 

Do you happen to use the fish shell? I can't run it either in that. But if I switch over to bash it works for me.

**Comment by bmsleight** [comment]
I am using bash.
I am  not getting the local webtools - See https://forum.espruino.com/conversations/399834/#comment17536853

**Comment by bmsleight** [comment]
Ready to merge ?

File: apps/exactwords/app.js [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+  var date = d.getDate();
+  var month = d.getMonth();
+
+  var timeStr = wordsFromTime(h,m);
+  var dateStr = wordsFromDayMonth(day, date, month);
+  
+  // draw time
+  // TO DO: Set Theme
+  g.setBgColor("#000000");
+  g.setColor("#ffffff");
```
Comment (thyttan) [inline apps/exactwords/app.js:196]: I think you could just do 

```suggestion
  g.setBgColor(g.theme.bg);
  g.setColor(g.theme.fg);
```

Here to make it compatible with dark/light (and other) system themes.

**Comment by bmsleight** [comment]
Thanks - nice improvement.

**Comment by thyttan** [comment]
Thanks for contributing! 🌸

**Comment by bmsleight** [comment]
woohoo!

**Pull request merged**